ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/EngineConnectorConfig.kt

```diff
@@ -75,7 +76,15 @@ interface EngineSSLConnectorConfig : EngineConnectorConfig {
     /**
      * Private key password provider
      */
-    val privateKeyPassword: () -> CharArray
+    public val privateKeyPassword: () -> CharArray
+
+    /**
+     * If not empty, the list of allowed CA certificates which will be used for client certificate authentication.
+     * If the list is empty, no client certification request will be requested.
+     *
+     * Currently only supported by Jetty Engine
+     */
+    val requiredClientCACertificates: List<X509Certificate>
```

Comment:
Why not trustStore(key store) with the certificates? With key store parameter, one can use same key store for both server and client auth, which is most common I believe as client certificates are self signed by server owners

Actually I have already made an implementation for Netty engine but chose to provide keyStore instead of list of certificates. I can contribute the implementation to this PR

Reply:
Good idea :)
Yes please. Help/contribution is always welcome.